docs/provider/daemon/fee/deposit_funds.md

```diff
@@ -0,0 +1,84 @@
+
+## Fragmenter
+
+The fragmenter is an interactive process that helps you splitting few UTXOs into many fragments of `5000` satoshis. It makes use of an ephemeral single-key wallet that is no longer used after the process is completed (or aborted).  
+
+To start the fragmenter for the Fee account run:
+
+```bash
+$ tdex fragmentfee
+# INFO[0000] send funds to address: el1qqtqj6psznm2km5axsj8qxgduzzznzjvmxfs3qh7h83hsp72hp0s5hzzwlzv92cgr44qtl4krrykddyv6xtcmvpusqraynmn0k
+# INFO[0000] Enter txid of fund(s) separated by a white space [press ENTER to skip or confirm]:
+```
+
+After generating and showing the ephemeral address, the command waits for you to prompt the txid of the funding tx(s).  
+Press _ENTER_ to confirm and continue the process: the tool calculates the optimal number of fragments based on the amount detected and sends the deposit fragments to the daemon's fee account.
+
+You can optionally start the fragmenter by specifying the max number of possible fragments it is allowed to create. By default it is set to 50 if not specified:
+
+```bash
+$ tdex fragmentfee --max_fragments 20
+```
+
+:::tip
+Take into account that the --max_fragments value basically defines the max number of outputs for the Fee account's deposit transaction.  
+In the Liquid network, transactions that weights more that 100K vBytes are rejected for being too heavy, therefore you should set this value at max to `70` or you'll risk to bump into this issue.
+:::
+
+If, for any reason, the process fails (like for example you pasted the wrong txid, or you forgot to unlock the wallet before this step) you can always resume it with:
+
+```bash
+$ tdex fragmentfee --txid <txid1> --txid <txid2> ...
+```
+
+The fragmenter is smart enough to recognize if any previous attempt exited before being completed. In that case, it expects you to resume that one by providing the list of funding txids. If this time everything's allright, the process will complete as described above. Only after a fragmentation process is completed, it is possible to go for another one.
+
+Another option is to abort the pending process instead of resuming it and send the "stuck" funds back to an address of yours instead of depositing them into the daemon HD wallet:
+
+```bash
+$ tdex fragmentfee --recover_funds_to_address <address>
+```
+
+This will send all the funds owned by the fragmenter to the specified address and abort the process instead of completing it.
+
+:::tip
+If you have funds stuck on the ephemeral wallet of the fragmenter, it is strongly suggested to backup your operator CLI datadir (usually located at `~/.tdex-operator` in Linux or `~/Library/Application\ Support/Tdex-operator in OSX) as long as you haven't completed or aborted the process, in order to prevent loosing them if you delete the datadir by accident.
+:::
```

Comment:
```suggestion
If you have funds stuck on the ephemeral wallet of the fragmenter, it is strongly suggested to backup your operator CLI datadir (usually located at `~/.tdex-operator` in Linux or `~/Library/Application\ Support/Tdex-operator in OSX) as long as you haven't completed or aborted the process, in order to prevent loosing them if you delete the datadir by accident.
:::
```